<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autorize</class>
 <widget class="QDialog" name="autorize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>281</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="LE_Login"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="LE_Password">
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="Check_Admin">
      <property name="text">
       <string>Администратор</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="Check_Director">
      <property name="text">
       <string>Директор</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="Check_Remember">
      <property name="text">
       <string>Запомнить меня</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="registration">
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="exit">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
